packages/testlab/package.json

```diff
@@ -28,7 +29,8 @@
     "should": "^13.1.3",
     "sinon": "^4.1.2",
     "supertest": "^3.0.0",
-    "swagger-parser": "^4.0.1"
+    "swagger-parser": "^4.0.1",
```

Comment:
Do we have any use for `swagger-parser` anymore?

Reply:
good catch nope